src/common/conf_file.cpp

```diff
@@ -119,20 +119,23 @@ int ConfFile::_parse_file(const char *addr, size_t len, const char *filename)
 
         // Make sure all casts to int are safe. No need to support options or sections
         // larger than INT_MAX
-        if (line_len > INT_MAX)
+        if (line_len > INT_MAX) {
             return -EINVAL;
+        }
```

Comment:
I find the addition { } verbose, but I can survive with them added. However here for example you removed the extra newline, that IMO makes the statements easier to read.

Reply:
I'm a really big fan of adding {} at every if statement. But you're right about the extra newline.